subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -35,6 +35,22 @@ The previous step will help you identify potential problems by issuing deprecati
 . Run `gradle wrapper --gradle-version {gradleVersion}` to update the project to {gradleVersion}.
 . Try to run the project and debug any errors using the <<troubleshooting.adoc#troubleshooting, Troubleshooting Guide>>.
 
+[[changes_8.2]]
+== Upgrading from 8.1 and earlier
+
+=== Deprecations
+
+[[compile_options_generated_sources_directory]]
+==== `link:{javadocPath}/org/gradle/api/tasks/compile/CompileOptions.html[CompileOptions]` method deprecations
+
+The following methods on `CompileOptions` are deprecated:
+
+- `getAnnotationProcessorGeneratedSourcesDirectory()`
```

Comment:
💭 It looks like the getter here has already been deprecated.  Maybe you should note this: "In addition to the already deprecated `getAnnotationProcessorGeneratedSourcesDirectory()` now `set...` and `...` are deprecated."

Reply:
It was deprecated in code but never mentioned in the upgrade guide. I am going to leave this as-is